### ISSUE TITLE
migrate repo and compendium and quickref to 2.13

### ISSUE THREAD

**Issue by bjornregnell** [lunduniversity/introprog#453]

**Comment by bjornregnell** [comment]
- [ ] `replace` instead of replaceAllLiterally
- [ ] `view.mapValues.toMap` instead of `mapValues` or something even better just using map as last attempt here:
```scala
scala> Vector(1,2,3,2,3).groupBy(x => x).mapValues(_.size)
                                         ^
       warning: method mapValues in trait MapOps is deprecated (since 2.13.0): Use .view.mapValues(f). A future version will include a strict version of this method (for now, .view.mapValues(f).toMap).
val res8: scala.collection.MapView[Int,Int] = MapView(<not computed>)

scala> Vector(1,2,3,2,3).groupBy(x => x).view.mapValues(_.size).toMap
val res9: Map[Int,Int] = Map(1 -> 1, 2 -> 2, 3 -> 2)

scala> Vector(1,2,3,2,3).groupBy(x => x).map(kv => kv._1 -> kv._2.size)
val res11: Map[Int,Int] = HashMap(1 -> 1, 2 -> 2, 3 -> 2)

```

**Comment by bjornregnell** [comment]
- [ ] Kill trait `Iterable`

**Comment by bjornregnell** [comment]
fixed by 635507fb31526515d653ec4694501f77b2a2c09a and 7795db7a095dd5c634f7533c65c45a3599acbafb and more
